web/src/components/PlanWorkoutModalForm/PlanWorkoutModalForm.js

```diff
@@ -0,0 +1,210 @@
+import {
+  Button,
+  Checkbox,
+  Divider,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Heading,
+  Input,
+  Select,
+  Spacer,
+} from '@chakra-ui/react'
+import { useForm } from 'react-hook-form'
+
+import { ActivityOptions } from '../ActivityOptions'
+
+export const PlanWorkoutModalForm = (props) => {
+  const { planWeek, planWorkout } = props
+  const form = useForm()
+  const {
+    handleSubmit,
+    register,
+    formState: { isSubmitting },
+  } = form
+  const onSubmit = (data) => {
+    data = {
+      ...data,
+      // this kinda stinks. Redwood forms handled this for us with `<HiddenField transformValue="Int">`.
+      //   with chakra forms we have to massage this ourselves.
+      planWeekId: parseInt(data.planWeekId),
```

Comment:
Noteworthy: redwood forms had the `transformValue` prop on fields to do this conversion; now we need to do that ourselves.